pkg/netceptor/conn.go

```diff
@@ -90,6 +92,7 @@ func (s *Netceptor) listen(ctx context.Context, service string, tlscfg *tls.Conf
 	s.Logger.Debug("%s added service %s to listener registry", s.nodeID, service)
 	s.listenerRegistry[service] = pc
 	cfg := &quic.Config{
+		Tracer:                  s.tracer,
```

Comment:
What happens if this is nil?

Reply:
Quic checks if tracer != nil and handles if nil. DefaultConnectionTracer returns nil if qlogDir is not defined: https://github.com/quic-go/quic-go/blob/master/qlog/qlog_dir.go#L41